docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -624,7 +738,57 @@ $ terraform apply
 ----
 In the terminal, Terraform displays a message that `rhoas_acl.acl.` will be created.
 . When you're ready to set your permissions, type *yes*.
-. (Optional) To delete the created resources, enter the following command:
+
+[id="proc-creating-kafka-topic-terraform_{context}"]
+==== Creating a Kafka topic
+
+[role="_abstract"]
+After you create a Kafka instance, you can create Kafka topics to start producing and consuming messages in your applications and services.
+
+.Prerequisites
+
+* You have an offline token that authenticates the Terraform resources with the {product-long-rhoas} API.
+* You have created a Kafka instance.
+
+.Procedure
+
+. To create a Kafka topic with default values, copy and paste the `rhoas_topic` resource shown in the following example into the `main.tf` file. This example uses the `topic` identifier and creates the `my-topic` Kafka topic. As you have already created the Kafka instance, Terraform can check dependencies for this new topic resource and knows the Kafka ID when you run this example resource.
```

Comment:
"Because you have..." instead of "As you have....".

See the entry for 'as' here: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a